benchmark/kvbench/test/custom_traffic_perftest.py

```
@@ -306,11 +614,16 @@ def _destroy_buffers(self):
-            self.world_size,
-        ), f"Matrix size is not the same as world size, got {tp.matrix.shape}, world_size={self.world_size}"
+        # Matrix size should be world * world (if matrix exists)
+        if tp.matrix is not None:
+            assert tp.matrix.shape == (
+                self.world_size,
+                self.world_size,
+            ), f"Matrix size is not the same as world size, got {tp.matrix.shape}, world_size={self.world_size}"
+        elif not tp.storage_ops:
+            raise ValueError(
+                "Traffic pattern must have either RDMA matrix or storage ops"
+            )
```

Ruff (0.15.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)